asyncapi/asyncapi.yml

```diff
@@ -319,202 +316,186 @@ channels:
     subscribe:
       message:
-        name: Passenger Announcement message
+        name: DpiPa
         schemaFormat: application/schema+json;version=draft-07
         payload:
-          $ref: json-schemas/dpi-pa.json
-        examples: [
-          { $ref: examples/dpi-pa_video.json },
-          { $ref: examples/dpi-pa_multiple.json },
-          { $ref: examples/dpi-pa_text.json },
-          { $ref: examples/dpi-pa_clear.json }
-        ]
+          $ref: json-schemas/pe/dpi/pa/dpi-pa.json
+        examples:
+          - $ref: json-schemas/pe/dpi/pa/dpi-pa_multiple.example.json
+          - $ref: json-schemas/pe/dpi/pa/dpi-pa_text.example.json
+          - $ref: json-schemas/pe/dpi/pa/dpi-pa_clear.example.json
+          - $ref: json-schemas/pe/dpi/pa/dpi-pa_video.example.json
         bindings:
           mqtt:
             qos: 1
             retain: true
   pe/dpi/arriving:
     description:
-      $ref: markdown/dpi-arriving.md
+      $ref: json-schemas/pe/dpi/arriving/dpi-arriving.md
     subscribe:
       message:
-        name: Arriving Message
+        name: DpiArriving
         schemaFormat: application/schema+json;version=draft-07
         payload:
-          $ref: json-schemas/dpi-arriving.json
-        examples: [
-          { $ref: examples/dpi-arriving.json }
-        ]
+          $ref: json-schemas/pe/dpi/arriving/dpi-arriving.json
+        examples:
+          - $ref: json-schemas/pe/dpi/arriving/dpi-arriving.example.json
         bindings:
           mqtt:
             qos: 1
             retain: false
   pe/dpi/connections:
     description:
-      $ref: markdown/dpi-connections.md
+      $ref: json-schemas/pe/dpi/connections/dpi-connections.md
     subscribe:
       message:
-        name: The Connections Schema
+        name: DpiConnections
         schemaFormat: application/schema+json;version=draft-07
         payload:
-          $ref: json-schemas/dpi-connections.json
-        examples: [
-          { $ref: examples/dpi-connections.json }
-        ]
+          $ref: json-schemas/pe/dpi/connections/dpi-connections.json
+        examples:
+          - $ref: json-schemas/pe/dpi/connections/dpi-connections.example.json
         bindings:
           mqtt:
             qos: 1
             retain: false
   pe/dpi/key_stops:
     description:
-      $ref: markdown/dpi-key_stops.md
+      $ref: json-schemas/pe/dpi/key_stops/dpi-key_stops.md
     subscribe:
       message:
-        name: Key Stops message
+        name: DpiKeyStops
         schemaFormat: application/schema+json;version=draft-07
         payload:
-          $ref: json-schemas/dpi-key_stops.json
-        examples: [
-          { $ref: examples/dpi-key_stops.json }
-        ]
+          $ref: json-schemas/pe/dpi/key_stops/dpi-key_stops.json
+        examples:
+          - $ref: json-schemas/pe/dpi/key_stops/dpi-key_stops.example.json
         bindings:
           mqtt:
             qos: 1
             retain: true
   pe/dpi_command:
```

Comment:
@eivindga should we change this to `pe/dpi/command`?

Reply:
yes